internal/server/query.go

```diff
@@ -223,4 +223,12 @@ const (
          where last_status_time > now() - interval '%d seconds'
            and operational_state = 'active';
 `
+
+	updateController = `
+		update server_controller
+		set address = @controller_address,
+			description = @controller_description,
+			update_time = now()
+		where private_id = @controller_private_id;
```

Comment:
```suggestion
		update server_controller
		   set address     = @controller_address,
			   description = @controller_description,
			   update_time = now()
		 where private_id = @controller_private_id;
```

As per our [sql style guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md#data-manipulation-statements) you want to align the keywords to form a "river".